packages/fedify/src/federation/middleware.test.ts

```diff
@@ -35,18 +38,7 @@ import {
 import { test } from "../testing/mod.ts";
 import { lookupObject } from "../vocab/lookup.ts";
 import { getTypeId } from "../vocab/type.ts";
-import {
-  Activity,
-  Announce,
-  Create,
-  type CryptographicKey,
-  Invite,
-  Multikey,
-  Note,
-  Object,
-  Offer,
-  Person,
-} from "../vocab/vocab.ts";
+import * as vocab from "../vocab/vocab.ts";
```

Comment:
If you're wondering why I had to make this change, I needed to use `Object.fromEntries` whilst debugging.